docs/source/daml/reference/structure.rst

```diff
@@ -53,7 +51,11 @@ template body
         Only create the contract if the conditions after ``ensure`` evaluate to true.
 
     :ref:`choices <daml-ref-choice-structure>`
-    	``controller nameOfParty can nameOfChoice``
+    	``controller nameOfParty can nameOfChoice do``
+
+        or
+
+        ``choice : () controller do``
```

Comment:
To me this somehow indicates that the return type is always `()`, but it can be anything. How about
```
choice NameOfChoice : ReturnType controller nameOfParty do
```
If we do this, we should probably change the summary of the old syntax to
```
controller nameOfParty can NameOfChoice : ReturnType do
```
There's also the option to have choice arguments following the `with` keyword but I guess we don't want that here since we don't have it for the old syntax.